README.md

```diff
@@ -92,7 +92,27 @@ You can use Docker instead of installing the gems yourself. Run `docker-compose
 Also you can run it as a standalone container like this:
 `docker run --rm codegram/decidim bundle exec bin/decidim <app-name>`.
 
-## Notes
+## How to contribute
 
-* You can use your own application layout - hooks have automatically been installed.
-* You can append your own `js` and `css`, files have automatically been replaced.
+In order to develop on decidim, you'll need:
+
+* **PostgreSQL** 9.4+
```

Comment:
I've removed that section as it was a poor man's version of the one we have now.